backend/src/main/java/org/cryptomator/hub/api/VaultResource.java

```diff
@@ -259,7 +259,7 @@ public String unlock(@PathParam("vaultId") UUID vaultId, @PathParam("deviceId")
 		var access = AccessToken.unlock(vaultId, deviceId, jwt.getSubject());
 		if (access != null) {
 			UnlockVaultEvent.log(jwt.getSubject(), vaultId, deviceId, UnlockVaultEvent.Result.SUCCESS);
-			return access.jwe;
+			return Response.ok(access.jwe).header("Cryptomator-Mobile-Access", license.isValid() ? "READ_WRITE" : "READ_ONLY").build();
```

Comment:
Can we even reach this point (unlock: status 200), if the license is not valid?

Reply:
Your remark made me think about it again and I guess my answer also made clear what the only distinction is: Check if there is a license set or not, see 325d050656758bd0659785e74fa4d4949a9ce85b. Checking validity is already done by the `ActiveLicense` filter.

Reply:
Regarding naming: Instead of naming the header after what it is used for, should we name it after its reason? Something like `Subscription-State` with values `ACTIVE` and `INACTIVE`

Reply:
Yeah, that's what I was thinking about as well but couldn't come up with a good name. I think I like `Hub-Subscription-State`! Made the changes in de5be39c2d37192c78eca4af69fde1d88306953e.